packages/sdk/src/client/edition/schedules.ts

```diff
@@ -187,15 +190,20 @@ export async function mintInfosFromMinter(
   const signerOrProvider = LazyPromise(() => expectSignerOrProvider().then((v) => v.signerOrProvider))
 
   const interfaceId = await idempotentCachedCall(`minter-interface-id-${minterAddress}`, async () => {
-    const minterModule = IMinterModule__factory.connect(minterAddress, await signerOrProvider)
+    const minterModule = IMinterModuleV2__factory.connect(minterAddress, await signerOrProvider)
 
     return (await minterModule.moduleInterfaceId()) as MinterInterfaceId
   })
 
   return Promise.all(
     mintIds.map(async (mintId) => {
+      if (HANDLED_MINTER_INTERFACE_IDS.indexOf(interfaceId) === -1) {
```

Comment:
handling this case at the top, so that we can then use exhaustiveGuard in the switch case below